### ISSUE TITLE
Multiple issues

### ISSUE THREAD

**Issue by kaby76** [kaby76/AntlrVSIX#30]
There are multiple issues with the extension, which is preventing me from releasing the next version:

* The build does not reliably work. I don't know why but VS 2019 sometimes add the Server/bin/Debug/... files to the .vsix file, sometimes it does not. I have found that it computes the file Client/obj/Debug/files.json sometimes incorrectly in the GenerateFileManifest step. It should be 61K long, but is 21K when it is done incorrectly. Timestamps, running as Administrator, etc does not reliably fix the problem. It's like it only appears after 10+ rebuilds.

* The server does not compute the refs and defs correctly. I am fixing this, but it's hard to get it to work when the first problem blocks fixes.

* The extension does not work when typing with keyboard. It reparses, but I think the server is slow.

**Comment by kaby76** [comment]
It seems that if I run the extension from an experimental version of VS2019 (via F5), typing response is very quick. If I install the extension, the response is very slow, and the focus is lost most of the time.

**Comment by kaby76** [comment]
There is a problem with the LASets computation.

**Comment by PrismElixir** [comment]
Re: Build Issue:
I've heard (and cannot find the page where I read it) that VS determines the VSIX contents on project load, not on builds/rebuilds.  So, the "workaround" is after making your changes, build as usual.  Then close the project. Now Reopen the project and Build (not rebuild).

Hope this helps.

**Comment by kaby76** [comment]
> Re: Build Issue:
> I've heard (and cannot find the page where I read it) that VS determines the VSIX contents on project load, not on builds/rebuilds. So, the "workaround" is after making your changes, build as usual. Then close the project. Now Reopen the project and Build (not rebuild).
> 
> Hope this helps.

Yep, I "discovered" this after trying all sorts of ways to fix the build. But, only after many hours, of course.
